src/index.js

```diff
@@ -80,6 +84,7 @@ export const litecoinCurrencyPluginFactory = makePluginFactory(litecoinInfo)
 // Legacy uppercased names:
 export {
   bitcoinCurrencyPluginFactory as BitcoinCurrencyPluginFactory,
+  decredCurrencyPluginFactory as DecredCurrencyPluginFactory,
```

Comment:
There is no reason to ever add anything to this list; we only keep it around for backwards-compatibility. Anybody using the uppercased names should just switch to the lowercase names.